<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmrActualizarLocal</class>
 <widget class="QDialog" name="FmrActualizarLocal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Actualizar Datos del Local</string>
  </property>
  <widget class="QPushButton" name="cmdGrabar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>20</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CmdCerrar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccion :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teDireccion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>22</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>nombre :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbDeshabilitado">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Deshabilitado</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbHabilitado">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Habilitado</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
